### PR TITLE
feat: add create origin command

### PR DESCRIPTION
**Pull request by maxwelbm**
- add create/origin command
- fix: resolved list edge application
- upgrade go version

### CODE REVIEW

File: cmd/root.go [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/*
```
Comment (PatrickMenoti) [inline cmd/root.go:1]: Did you mean to commit this file?

File: main.go [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/*
```
Comment (PatrickMenoti) [inline main.go:1]: Did you mean to commit this file?

File: cmd/root.go (hunk 1 above)
Comment (maxwelbm) [reply cmd/root.go:1]: I wasn't testing cobra-cli :face_exhaling:

File: main.go (hunk 2 above)
Comment (maxwelbm) [reply main.go:1]: I wasn't testing cobra-cli

**Review by PatrickMenoti — APPROVED** [review]
We just need to decide how we handle the messages package!!!

**Pull request merged**